form comleted

AlexPro100pro opened the pull request:
[DEMO LINK](https://AlexPro100pro.github.io/layout_html-form/)
[TEST REPORT LINK](https://AlexPro100pro.github.io/layout_html-form/report/html_report/)

alexandra-protyanova commented:
https://user-images.githubusercontent.com/102182808/206989438-4a52e3b3-24bd-4a18-89b5-6241e1524544.mov

There are big differences  between the mockup and your demo, let us fix it 😉

alexandra-protyanova commented on src/index.html:
```diff
@@ -11,7 +11,193 @@
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get">
```
```suggestion
    <form
      class="form"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="get"
    >
```

alexandra-protyanova replied:
[CHECKLIST ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) point 3

alexandra-protyanova commented on src/index.html:
```diff
@@ -11,7 +11,193 @@
+      method="get">
+      <fieldset class="form-block top">
+        <legend>Personal Information</legend>
+        <div class="form-field top">
```
divs are redundant in this task

alexandra-protyanova replied:
if you use them to put each input on new line, use `display: block` instead

AlexPro100pro replied:
i cant use display: block, only margins ( checklist 10 )
![image](https://user-images.githubusercontent.com/37651667/207048481-3c1a3599-71c8-4d1f-aea4-661a1064c13f.png)
because of that I didn't use this the build `<label> <input> </label>`

alexandra-protyanova replied:
okay, you can use tag `br` then

alexandra-protyanova commented on src/style.css:
```diff
@@ -1 +1,20 @@
+  margin: 20px 0;
+}
+
+.top {
```
use `last-child` to cancel bottom margins to last inputs and blocks

AlexPro100pro commented:
Hi, this work would be approved. I fixed problem with aprove this work in mate platform, i deleted my old fork and did new fork with old files. I taged Artur, he should be aproved it.

AlexPro100pro commented:
i can`t see your video, player don`t work. i know that work doesnt pixel perfect, but it wasnt critical
